packages/mangrove/src/core.ts

```diff
@@ -68,3 +68,5 @@ export type OfferFailReason =
   | "mgv/makerAbort"
   | "mgv/makerTransferFail"
   | "mgv/makerReceiveFail";
+export type Date = number;
```

Comment:
It's not clear how exactly it is decoded as a number. Is it a timestamp? We should probably name it `TimestampMs` then

Reply:
Timestamp (expressed in seconds since unix epoch) beyond which the order is no longer valid, 0 means forever.
Renamed it to `Timestamp` and added comment